lib/shelley/src/Cardano/Wallet/Shelley/Transaction.hs

```diff
@@ -1249,72 +1256,119 @@ mkUnsignedTx
     -> [(Cardano.StakeAddress, Cardano.Lovelace)]
     -> [Cardano.Certificate]
     -> Cardano.Lovelace
+    -> Maybe (TokenMap, [Script KeyHash])
```

Comment:
Just realised this means I can only support minting, not burning. I'll need to fix this up.

Reply:
Fixed in cb8f694.